Cargo.toml

```diff
@@ -11,6 +11,9 @@ authors = ["Richard Newman <rnewman@twinql.com>", "Nicholas Alexander <nalexande
 [dev-dependencies.datomish-cli]
   path = "cli"
 
+[dev-dependencies.datomish-explorer]
+  path = "explorer"
+
 [[bin]]
 name = "datomish-cli"
```

Comment:
We could also make this the default bin, since the current cli doesn't do anything.  Then it'd just be `cargo run`